script-generator/src/main/java/com/coolbitx/wallet/signing/scriptlib/VetScript.java

```diff
@@ -0,0 +1,284 @@
+package com.coolbitx.wallet.signing.scriptlib;
+
+import com.coolbitx.wallet.signing.utils.ScriptArgumentComposer;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler;
+import com.coolbitx.wallet.signing.utils.ScriptData;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.HashType;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.SignType;
+import com.coolbitx.wallet.signing.utils.ScriptData.Buffer;
+
+import static com.coolbitx.wallet.signing.utils.ScriptAssembler.TYPE_RLP;
+
+import com.coolbitx.wallet.signing.utils.HexUtil;
+
+public class VetScript {
+  
+  public static void listAll() {
+    System.out.println("Vet: \n" + getVetTransactionScript() + "\n");
+    System.out.println("VTHO: \n" + getVthoTokenScript() + "\n");
+    System.out.println("VIP191 Origin: \n" + getVIP191OriginScript() + "\n");
+    System.out.println("VIP191 Vet Cert: \n" + getVetCertificateScript() + "\n");
+  }
+
+  public static String getVetTransactionScript() {
+
+    /*
+     * chainTag - 1byte - rlp
+     * blockRef - 8 byte - fixed length
+     * expiration - 4 byte - rlp
+     * clauses - array
+     * to - 20 byte - fixed length (nullable)
+     * value - 32 byte - rlp
+     * data - no mention (blob kind)
+     * gasPriceCoef - 1 byte - rlp
+     * gas - 8 byte - rlp
+     * dependsOn - 32 byte - fixed length (nullable)
+     * Nonce 8 byte - rlp
+     */
+    ScriptArgumentComposer sac = new ScriptArgumentComposer();
+    ScriptData argChainTag = sac.getArgumentRightJustified(1);
+    ScriptData argBlockRef = sac.getArgumentRightJustified(8);
+    ScriptData argExpiration = sac.getArgumentRightJustified(4);
+    ScriptData argTo = sac.getArgument(20);
+    ScriptData argValue = sac.getArgumentRightJustified(32);
+    ScriptData argData = sac.getArgumentRightJustified(6);
+    ScriptData argGasPrice = sac.getArgumentRightJustified(1);
+    ScriptData argGas = sac.getArgumentRightJustified(8);
+    ScriptData argDependsOn = sac.getArgumentRightJustified(32);
+    ScriptData argNonce = sac.getArgumentRightJustified(8);
+
+    String script = new ScriptAssembler()
+        // set coinType to 0332
+        .setCoinType(0x0332)
+        .arrayPointer()
+        // chainTag
+        .rlpString(argChainTag)
```

Comment:
Seems that `ChainTag` can be hard-coded in `4a`, cause CoolWallet only need to support main-net.

Reply:
@hank121314 changes had been committed.